modules/spock/src/main/groovy/org/testcontainers/spock/SpockTestDescription.groovy

```diff
@@ -0,0 +1,36 @@
+package org.testcontainers.spock
+
+import groovy.transform.PackageScope
+import org.spockframework.runtime.extension.IMethodInvocation
+import org.testcontainers.lifecycle.TestDescription
+
+/**
+ * Spock specific implementation of a Testcontainers TestDescription.
+ *
+ * Filesystem friendly name is based on Specification and Feature.
+ */
+@PackageScope
+class SpockTestDescription implements TestDescription {
+
+    String specName
+    String featureName
+
+    static SpockTestDescription fromTestDescription(IMethodInvocation invocation) {
+        return new SpockTestDescription([
+            specName: invocation.spec.name,
+            featureName: invocation.feature.name
+        ])
+    }
+
+    @Override
+    String getTestId() {
+        return null
+    }
+
+    @Override
+    String getFilesystemFriendlyName() {
+        return [specName, featureName].collect {
```

Comment:
AFAIK `specName` can be an arbitrary string. Perhaps we should encode it (similar to #1326)

Reply:
Yes, both actually (since Groovy method names can be arbitrary Strings?).